go/ct/spc/specification.go

```diff
@@ -1138,6 +1138,31 @@ func getAllRules() []Rule {
 		},
 	}...)
 
+	// --- BLOBBASEFEE ---
+
+	rules = append(rules, rulesFor(instruction{
+		op:         BLOBBASEFEE,
+		staticGas:  2,
+		pops:       0,
+		pushes:     1,
+		conditions: []Condition{IsRevision(R13_Cancun)},
```

Comment:
```suggestion
		conditions: []Condition{RevisionBounds(R13_Cancun, NewestSupportedRevision)},
```

does this makes sense to you?